asm/macros/scrcmd.inc

```diff
@@ -559,10 +563,10 @@
     .short 104
     .endm
 
-    .macro ScrCmd_069 arg0, arg1
+    .macro GetPlayerXZPos destVarIDX, destVarIDZ
```

Comment:
**question:** Is this still clear as `GetPlayerPos`?

Reply:
There's a different script command (currently `ScrCmd_317`) that returns X, Y, and Z which is why I went more specific. FWIW, DSPRE names them:
- XZ - GetPlayerPosition
- XYZ - GetPlayerPositionFull

But personally I think those names are a little too ambiguous.

Reply:
After discussion on discord, we landed on:
- Retrieves XZ - `GetPlayerMapPos`
- Retrieves XYZ - `GetPlayer3DPos`